README.md

```diff
@@ -0,0 +1,79 @@
+# Kiwix Apple Custom Apps
+
+Kiwix Apple custom apps are iOS/macOS apps running [Kiwix for
+Apple](https://github.com/kiwix/apple) against a
+pre-configured ZIM file.
+
+This project contains data and scripts needed to create specific
+ custom Kiwix Apple apps.
+
+[![CodeFactor](https://www.codefactor.io/repository/github/kiwix/kiwix-apple-custom/badge)](https://www.codefactor.io/repository/github/kiwix/kiwix-apple-custom)
+[![CI Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/kiwix/apple/actions/workflows/ci.yml?query=branch%3Amain)
+[![CD Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd.yml/badge.svg?branch=main)](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd?query=branch%3Amain)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Custom app folder
+
+[In the repository](https://github.com/kiwix/kiwix-apple-custom),
+each custom configuration is isolated in a so called custom app
+folder. If you need to create a new one for a new custom app, please use a folder name
+that is lowercased and contains no space.
+
+## `info.json` file
+
+The configuration of the custom app is handled using the `info.json`
+file which **is required to be placed in the custom app folder**. Take example on an already
+existing one if you need to create a new custom app. 
+
+### The required fields are:
+- `about_app_url` - this is an external link that is placed in the "About section" of the application. (Eg. "https://www.dwds.de")
+- `about_text` - this is a custom text that is placed in the "About section" describing what the application is about. It is not supporting html tags, but new lines can be added with '\n'.
+- `app_name` - Name of the app, as it will appear on device, and in App Store
+- `app_store_id` - this should to be taken from the developer.apple.com, where the application release is prepared. Note you can use the app_store_id even if the app is not yet released. The id is used within the app in the "Rate the app" section, so users can be redirected to a specific app in the App Store, to rate it.
+- `build_version` - it should be a single digit number, used as the last part of the final app version.
```

Comment:
I am not so sure about this one now to be honest. 
We overwrite this value from the release/tag as part of the CD process. 
It's hard to explain that we need this "just in case", or that we need this if we want to release all apps at once, which we do not fully have yet.
Maybe we can remove this and have a fallback value instead, like fallback to 0 (zero)?
What do you think?

Reply:
Yes, I understood from Friday's discussion that we're now tag-based completely so this should be removed.

Reply:
I did the removal as part of the wikimed branch, since that has both (dwds and wikimed) info.json files:
https://github.com/kiwix/kiwix-apple-custom/pull/45

Reply:
I've removed this section, and added the versioning and releasing part, to make it clear how it all fits together.